cmd/romana.k8s

```diff
@@ -0,0 +1,403 @@
+}
+
+show_owner() {
+	# Without arguments, show all the owners
+	if (( $# == 0 )); then
+		show_owners
+		return
+	fi
+
+	# Show usage.
+	# TODO: What if the owner's name is 'help' ?
+	if [[ "$1" == "help" ]]; then
+		echo "romana show-owner [owner-name...]"
+		return
+	fi
+
+	# Get a list of all the owners, save the data
+	if ! response=$(curl -s -f -H "Content-Type: application/json" "${tenant_url}${tenant_path}"); then
+		echo "Unable to fetch owners using URL '${tenant_url}${tenant_path}'"
+		return
+	fi
+
+	# For each owner name...
+	for i in "$@"; do
+		# Find the owner with that name
+		len=$(jq -r --arg name "$i" '.[] | select(.Name==$name) | length' <<< "$response")
+		if (( len == 0 )); then
+			echo "Unable to find tenant '$project' ($i)"
+			continue
+		fi
+
+		read -r id _ < <(jq -r --arg name "$i" '.[] | select(.Name==$name) | "\(.Id)"' <<< "$response")
+
+		# Get the list of tiers
+		if ! tier_list=$(curl -s -f -H "Content-Type: application/json" "${tenant_url}${tenant_path}/${id}/segments"); then
+			echo "Unable to list tiers for owner '$i'"
+			continue
+		fi
+
+		# Write the data
+		printf "owner:%-32s\n" "$i"
+		printf "tiers: "
+		sep=""
+		while read -r tier_name _; do
+			printf "%s%s" "$sep" "$tier_name"
+			sep=", "
+		done < <(jq -r '.[].Name' <<< "$tier_list")
+		printf "\n"
+	done
+}
+
+show_owners() {
+	# Get the list of owners
+	if ! response=$(curl -s -f -H "Content-Type: application/json" "${tenant_url}${tenant_path}"); then
+		echo "Unable to fetch owners using URL '${tenant_url}${tenant_path}'"
+		return
+	fi
+	# Determine the number of owners
+	num_owners=$(jq 'length' <<< "$response")
+	if (( num_owners == 0 )); then
+		echo "No owners found."
+		return
+	fi
+	# List the owners
+	printf "Listing %d owner(s)\n" "$num_owners"
+	while read -r owner _; do
+		printf "%-32s\n" "$owner"
+	done < <(jq -r '.[].Name' <<< "$response")
+}
+
+add_tier() {
+	if (( $# == 0 )) || [[ "$1" == "help" ]]; then
+		echo "romana add-tier <owner-name> <segment-name>"
+		return
+	fi
+	if (( $# != 2 )); then
+		echo "ERROR: expected 2 parameters, received $#"
+		return
+	fi
+	local owner_name="$1" tier_name="$2"
+	if ! response=$(curl -s -f -H "Content-Type: application/json" "${tenant_url}${tenant_path}"); then
+		echo "Unable to fetch owners using URL '${tenant_url}${tenant_path}'"
+		return
+	fi
+	read -r owner_id _ < <(jq -r --arg name "$owner_name" '.[] | select(.Name==$name) | "\(.Id)"' <<< "$response")
+
+	if ! [[ "$owner_id" ]]; then
+		echo "Unable to find owner id using owner ref '$owner_name'"
+		return
+	fi
+
+	post_data=$(printf '{"id": 0, "name": "%s"}' "$tier_name")
+	tier_path="$tenant_path/$owner_id/segments"
+	echo "Sending request to '${tenant_url}${tier_path}': '$post_data'"
+	curl -s -X POST -H "Content-Type: application/json" --data "$post_data" "${tenant_url}${tier_path}"; printf "\n"
+}
+
+add_policy() {
```

Comment:
Do we really want to offer a tool to add policies? Shouldn't the user just use the K8S API for that in some documented way (curl)? I know it's a little easier with our tool, but it also makes it look like these are special policies, when they are really nothing but ordinary K8S policies.